elastic/logs/tasks/field-caps-non-empty-fields.json

```diff
@@ -2,10 +2,9 @@
   "name": "field-caps-exclude-empty-fields",
   "operation": {
     "operation-type": "field-caps",
-    "index": "auditbeat*",
-    "request-params": {
-      "include_empty_fields": "false"
-    }
+    "base-operation-type": "raw-request",
```

Comment:
The `base-operation-type` parameter makes only sense with a custom runner referenced through `"operation-type": "multi-cluster-wrapper`. The purpose of it is to execute the base operation on all clusters provided for Rally. I don't think we need it in this challenge, so I would simplify to `"operation-type": "raw-request"`.